app/controllers/admin/content/translations.js

```diff
@@ -0,0 +1,20 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+  isLoading: false,
+
+  actions: {
+    translationsDownload() {
+      this.set('isLoading', true);
+      this.get('loader')
+        .load('/admin/content/translations/all')
```

Comment:
@mrsaicharan1 Use the service I defined in https://github.com/fossasia/open-event-frontend/pull/2540
here. 
```js
this.get('loader')
        .downloadFile('/admin/content/translations/all')
        .then(()=> {
         this.get('notify').success(this.get('l10n').t('Translations Zip generated successfully.'));
          })

Reply:
![Screenshot from 2019-04-09 18-23-13](https://user-images.githubusercontent.com/25197147/55780824-77b6bd80-5ac6-11e9-9fa8-51cc2757658d.png)
The Downloder service helped a lot! It's working perfectly now :)

Reply:
The downloader service helped a lot! It's working perfectly now :)
![Screenshot from 2019-04-09 18-24-44](https://user-images.githubusercontent.com/25197147/55780950-bb112c00-5ac6-11e9-9ed8-ead98f0a5fce.png)